### ISSUE TITLE
negative number condition in when block formatted incorrectly

### ISSUE THREAD

**Issue by yshrsmz** [shyiko/ktlint#160]
since 0.16.0

before
```kotlin
when (foo){
  0 -> {
  }
  1 -> {
  }
  -1 -> {
  }
}
```

after
```kotlin
when (foo){
  0 -> {
  }
  1 -> {
  } -
  1 -> {
  }
}
```

**Comment by shyiko** [comment]
Thank you for the ticket. Fixed in [0.16.1](https://github.com/shyiko/ktlint/blob/master/CHANGELOG.md#0161---2018-02-27). It should become available through Maven Central within 25 minutes.

**Comment by Tolriq** [comment]
@shyiko 
Tested with 0.16.1 now it produces:
```
catch (e: MalformedURLException) { -
                100
            }
```

**Comment by shyiko** [comment]
Thanks for the heads up, @Tolriq. Fixed in [0.17.0](https://github.com/shyiko/ktlint/releases/tag/0.17.0) (Maven Central sync might take up to 25 minutes).
